projects/plugins/jetpack/class.jetpack.php

```diff
@@ -6200,19 +5568,6 @@ public static function check_identity_crisis() {
 		return Jetpack_Options::get_option( 'sync_error_idc' );
 	}
 
-	/**
-	 * Checks whether the home and siteurl specifically are allowed.
-	 * Written so that we don't have re-check $key and $value params every time
-	 * we want to check if this site is allowed, for example in footer.php
-	 *
-	 * @since  3.8.0
-	 * @return bool True = already allowed False = not on the allowed list.
-	 */
-	public static function is_staging_site() {
```

Comment:
This function is called by the following plugins in the w.org repo:

- [ ] `connect-for-woocommerce`: classes/class-wc-connect-api-client.php,409
- [ ] `connect-for-woocommerce`: classes/class-wc-connect-help-view.php,129
- [ ] `easyreservations`: includes/class-er-tracker.php,105
- [ ] `geodirectory`: includes/admin/class-geodir-admin-tracker.php,114
- [ ] `woocommerce`: includes/class-wc-tracker.php,108
- [ ] `woocommerce-services`: classes/class-wc-connect-jetpack.php,72

Reply:
- `connect-for-woocommerce` -- Old a8c plugin that is closed. only 1 outdated JP install with it. Going to mark this one fine.
- `easyreservations` -- Same as Woo below. New way, then deprecated with proper gating.
- `geodirectory` -- [Uses the old way, but has a proper gate](https://plugins.trac.wordpress.org/browser/geodirectory/trunk/includes/admin/class-geodir-admin-tracker.php#L113). **We should reach out to encourage them to update. Under active development.**
- `woocommerce` -- Will return the new way first and does verify that `is_staging_site` is a callable before using it worst case.
- `woocommerce-services` -- Same as above. New way first, the deprecated as a backup for old versions with a proper check.

Reply:
`geodirectory` -- Left message https://wordpress.org/support/topic/update-jetpack-function-2/